ivy/functional/backends/torch/transformations.py

```diff
@@ -0,0 +1,17 @@
+# global
+import functorch
+from typing import Callable, Union, Sequence, Optional
+
+# local
+import ivy
+
+
+def vmap(func: Callable,
+         in_axes: Union[int, Sequence[int], Sequence[None]] = 0,
+         out_axes: Optional[int] = 0) -> Callable:
+    def _vmap(*args):
+        new_fun = lambda *args: ivy.to_native(func(*args)) 
+        new_func = functorch.vmap(new_fun, in_axes, out_axes)
```

Comment:
is this not natively supported in `torch` now? Or maybe this is only a partial version?

Reply:
At the moment, torch only has vmap in the nightly versions as they consider it a prototype. The team has made `functorch` as an extension of torch for functional programming. In [this Issue](https://github.com/pytorch/pytorch/issues/42368) on Pytorch repo, they referrerd to functorch. It seems `vmap` might be incorporated into `torch` stable versions when it is fully functional in `functorch`.